### DISCUSSION TITLE
How to transfer element values from multiapps to master?

### DISCUSSION THREAD

**Discussion by 1791445402** [idaholab/moose#24449] [Q&A General]
Hello developers

I currently have a master and multiple multiapps, and l need to pass the element value from one of the multiapps to the maste. However, when I used type = MultiAppNearestNodeTransfer in the [transfers] of the master, I found that the element value cannot be effectively passed, but when l change the element value to another node value, the value can be effectively passed.

lt seems that type = MultiAppNearestNodeTransfer can only pass node values. Does anyone know how to pass element values?
```
master.i
[Transfers]
  [A_from_multi]
    type = MultiAppNearestNodeTransfer
    direction = from_multiapp
    multi_app = multi_A
    source_variable = A
    variable = A
  []
[]
```

**Comment by GiudGiud** [comment]
Hello

MultiAppNearestNodeTransfer was not well named. 
It actually defaults to passing the element centroid value for Constant Monomial variables. 
It's on my to-do list to rename it. 

Guillaume
